syscall/include/syscall/vcpu.h

```diff
@@ -46,8 +46,7 @@ namespace syscall
     ///   @return Does not return on success. Returns a bsl::errc_type
     ///     on failure.
     ///
-    extern "C" [[nodiscard]] bsl::errc_type
-    run_vcpu(vcpu_context * const vc) noexcept;
+    extern "C" [[nodiscard]] bsl::errc_type run_vcpu(vcpu_context *const vc) noexcept;
```

Comment:
The const is redundant on the right side of the * since vc is on the stack. Did you mean `vcpu_context const *`?